templates/institutional/institution-landing-en.html

```diff
@@ -0,0 +1,182 @@
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+			<div class="col-md-4">
+				<h3><a href="#">[Hyperlink text]</a></h3>
+				<p>Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.</p>
+			</div>
+		</div>
+	</section>
+</div>
+
+<div class="well well-sm provisional well-bold brdr-0 mrgn-bttm-0">
+	<div class="container">
+		<section>
+			<h2 class="h3 mrgn-tp-md">Contact us</h2>
+			<ul class="wb-eqht list-unstyled mrgn-tp-lg lst-spcd-2 colcount-md-3">
+				<li><a href="#">Contact [Institution]</a></li>
+				<li><a href="#">[Top contact task 2]</a></li>
+				<li><a href="#">[Top contact task 3]</a></li>
+			</ul>
+		</section>
+		<div class="row">
+			<section class="col-md-8 pull-left mrgn-bttm-lg">
+				<h2 class="h3 mrgn-tp-md">About the [Institution]</h2>
+				<ul class="list-unstyled mrgn-tp-lg lst-spcd-2 colcount-md-2">
+					<li><a href="#">Mandate</a></li>
+					<li><a href="#">Organizational structure</a></li>
+					<li><a href="#">Transparency</a></li>
+					<li><a href="#">Job opportunities</a></li>
+					<li><a href="#">Reports</a></li>
+					<li><a href="#">Compliance</a></li>
+					<li><a href="#">Enforcement notifications</a></li>
+					<li><a href="#">[Institutional link]</a></li>
+					<li><a href="#">[Institutional link]</a></li>
+					<li>More: <a href="#">About the [Institution]</a></li>
+				</ul>
+			</section>
+			<div class="col-xs-12 col-md-4 col-sm-5 pull-right">
+				<section class="lnkbx mrgn-bttm-lg">
+					<h2 class="h3">Minister</h2>
+					<p>
+						<a href="#">[(Honourable) first and last name]</a><br>
+						<small>Minister's official title</small>
+					</p>
+					<p>
+						<a href="#">[(Honourable) first and last name]</a><br>
+						<small>Minister's official title</small>
+					</p>
+					<p>
+						<a href="#">[(Honourable) first and last name]</a><br>
+						<small>Minister's official title</small>
+					</p>
+				</section>
+				{%- include components/gc-follow-us.html -%}
+			</div>
+		</div>
+	</div>
+</div>
+
+<div class="container">
+	<div class="row">
+		<section class="provisional wb-feeds gc-feeds limit-3 col-md-4">
+			<h2 class="h3">News</h2>
+			<ul class="feeds-cont list-unstyled lst-spcd">
+				<li>
+					<a data-ajax="../../components/feeds/demo/example-en.atom.json" href="#">
+					[Institution] news
+					</a>
+				</li>
+			</ul>
+			<p>More: <a href="#">[Institution] news</a></p>
+		</section>
+		<section class="col-md-8 gc-features">
+			<h2 class="h3">Features</h2>
+			<div class="row wb-eqht">
+				<div class="col-sm-6">
+					<div class="well well-sm brdr-rds-0 eqht-trgt">
+						<img class="img-responsive full-width" src="../img/news-360x203.png" alt="" />
```

Comment:
This image is missing in this PR. Maybe you want to link to feature component image at: `components/gc-features/img/feature-360x203.png`

Reply:
@duboisp Should we begin using this instead: https://via.placeholder.com/360x203/000000/FFFFFF.png

Reply:
Would it make more sens having one folders for all images instead of having images at various locations?

Reply:
@Ricokola By using the Placeholder consistently, we'll be able to technically do just that, plus it removes the overall size of GCWeb by not uploading bunch of images with numbers on them. So these two feature images will be the first ones to use the placeholder website to load place holder images but more templates and/or components will eventually follow.